sklearn/model_selection/tests/test_validation.py

```diff
@@ -2082,37 +2082,6 @@ def test_fit_and_score_failing():
     y = np.ones(9)
     fit_and_score_args = [failing_clf, X, None, dict(), None, None, 0, None, None]
     # passing error score to trigger the warning message
-    fit_and_score_kwargs = {"error_score": 0}
```

Comment:
The check on the warnings were moved below to `test_cross_validate_failing_fits_warnings` since `_fit_and_score` does not warn anymore